packages/ui/src/kit/VRadio/VRadio.vue

```diff
@@ -1,23 +1,29 @@
 <template>
-  <label :class="vRadioCssClass">
-    <input
-      v-bind="$attrs"
-      :checked="value === modelValue"
-      type="radio"
-      class="v-radio-input"
-      @input="onInputChange"
+  <div :class="vRadioCssClass">
+    <label-molecule
```

Comment:
please do not use `<label-molecule/>` for radio instead of `<label>`
if you need add tooltip to radios, use `<hover-atom>` with `<tooltip-atom>` atoms as it was defined in `label-molecule`